core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -95,11 +96,15 @@ class LoadBalancerService(config: WhiskConfig, verbosity: LogLevel)(
                 info(this, s"Completion counts: [${invokerCounts.mkString(", ")}]")
                 info(this, s"Invoker health: [${health.mkString(", ")}]")
             }
-            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth()) ++
-                getUserActivationCounts()
+            Map(LoadBalancerKeys.invokerHealth -> getInvokerHealth())
         })
 
     /**
+     * A bridge method to impedance match against RestAPI.
+     */
+    def acceptRequest(lbr: LoadBalancerReq) = publish(lbr._1)(lbr._2)
```

Comment:
It's a no go to get rid of the bridge methods.  They are in fact needed due to unit testing of the controller by itself (ControllerTestCommon) when Kafka and DB are not even up.   I am reverting aspects of the last commit so that it's slightly cleaner but without getting rid of these two methods.